src/Uno.UWP/Storage/Pickers/FileOpenPicker.iOS.cs

```diff
@@ -10,6 +10,8 @@
 using Foundation;
 using Windows.ApplicationModel.Core;
 using Uno.Helpers.Theming;
+using CoreAudioKit;
+using System.Xml.Linq;
```

Comment:
This too is probably not used.

Reply:
Just removed them, the keep being added at each build.